client/printing/print_grpc_pb.js

```diff
@@ -4,6 +4,17 @@
 var grpc = require('grpc');
 var print_pb = require('./print_pb.js');
```

Comment:
while you're working on this can you change instances of var over to `const` or `let`

Reply:
i mentioned this before, this is auto generated code. nobody should touch this

Reply:
ahhh sorry, i think i missed it